test/unit-tests/testAnalyticsFramework.js

```diff
@@ -1,6 +1,5 @@
 
-describe('Analytics Framework Unit Tests', function()
-{
+describe('Analytics Framework Unit Tests', function() {
```

Comment:
I think your auto formatter went crazy and moved all the '{' to the line above. Can you undo it?